tests/integration/helpers/not-test.js

```diff
@@ -9,7 +9,9 @@ module('Integration | Helper | not', function (hooks) {
   test('it renders', async function (assert) {
     this.set('inputValue', 'truthy');
 
-    await render(hbs`{{if (not this.inputValue) "YES" "NO"}}`);
+    await render(
+      hbs`{{!-- template-lint-disable no-negated-condition --}}{{if (not this.inputValue) "YES" "NO"}}`,
```

Comment:
to satisfy ember-template-lint. We do want the negated condition here because that's explicitly what this test is testing.